test/spec/controllers/eventRegistrations.spec.js

```diff
@@ -104,4 +104,15 @@ describe('Controller: eventRegistrations', function() {
     scope.registerUser();
     expect(openModal).toHaveBeenCalled();
   });
+
+  it('refreshRegistrations should assign reported attribute to registarnts', function() {
```

Comment:
Type in test description
```suggestion
  it('refreshRegistrations should assign reported attribute to registrants', () => {
```